openpype/hosts/celaction/hooks/pre_celaction_setup.py

```diff
@@ -0,0 +1,138 @@
+import os
+import shutil
+import subprocess
+import winreg
+from openpype.lib import PreLaunchHook, get_openpype_execute_args
+from openpype.hosts.celaction import api as caapi
+
+CELACTION_API_DIR = os.path.dirname(
+    os.path.abspath(caapi.__file__)
+)
+
+
+class CelactionPrelaunchHook(PreLaunchHook):
+    """
+    Bootstrap celacion with pype
+    """
+    workfile_ext = "scn"
+    app_groups = ["celaction"]
+    platforms = ["windows"]
+
+    def execute(self):
+        project_doc = self.data["project_doc"]
+        width = project_doc["data"]["resolutionWidth"]
+        height = project_doc["data"]["resolutionHeight"]
+
+        # Add workfile path to launch arguments
+        workfile_path = self.workfile_path()
+        if workfile_path:
+            self.launch_context.launch_args.append(workfile_path)
+
+        # setting output parameters
+        path_user_settings = "\\".join([
+            "Software", "CelAction", "CelAction2D", "User Settings"
+        ])
+        winreg.CreateKey(winreg.HKEY_CURRENT_USER, path_user_settings)
+        hKey = winreg.OpenKey(
+            winreg.HKEY_CURRENT_USER, path_user_settings, 0,
+            winreg.KEY_ALL_ACCESS
+        )
+
+        path_to_cli = os.path.join(CELACTION_API_DIR, "cli.py")
```

Comment:
Rather don't import anything from `api` (It should be used only inside DCC or "host" process). If it's not the case the it probably should not be inside `api` folder.
```suggestion
        path_to_cli = os.path.join(
            CELACTION_ROOT_DIR, "api", "cli.py"
        )
```